### ISSUE TITLE
Is this project abandoned? 

### ISSUE THREAD

**Issue by timothymarois** [dsmorse/gridster.js#147]
It appears there hasn't been much movement here in a long time and wondering if we need to fork this again? 4th fork? 

Is there a better alternative for the reason for discontinuing the project updates?

**Comment by dsmorse** [comment]
yes, I have retired from front end dev:  https://github.com/dsmorse/gridster.js/issues/97
